source/core/index.ts

```diff
@@ -23,6 +23,7 @@ import calculateRetryDelay from './calculate-retry-delay.js';
 import Options, {OptionsError, OptionsInit} from './options.js';
 import {isResponseOk, Response} from './response.js';
 import isClientRequest from './utils/is-client-request.js';
+import isUnixSocketURL from './utils/is-unix-url.js';
```

Comment:
Super nitpick, but you know me..

```suggestion
import isUnixSocketURL from './utils/is-unix-socket-url.js';
```

Reply:
PTAL